### DISCUSSION TITLE
Grand Orgue Control  Add Custom Panels,  Redesign or Reformat

### DISCUSSION THREAD

**Discussion by alex-aotearoa** [GrandOrgue/grandorgue#1943] [Help]
Is there a guide somewhere on how to build / add / reformat control panels for Grand Orgue, in particular Piotr Grabowskis Friesach. I would like to make simple panels that look more like the various simple examples as in the Pitea organ.  The reason being Ive already built my console with switches then I discovered Friesach.  I also find the Freisach console a bit hard to find my way around. Ive Read the excellent "The GrandOrgue Architecture for Sample-set Designers – A Tutorial" but it seems to leave that part out. thank you.

**Comment by larspalo** [comment]
You might benefit from using https://github.com/GrandOrgue/GoOdf but make sure to read the README.md with all the warnings first.

**Reply by jason-barba** [reply]
By studying details in the Friesach ODF you can at least  get a glimpse of how it could be done. Think object programming, that helps.
